src/components/Auth/ResetPassword.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import { useState, useCallback } from 'react';
+import { useHistory } from 'react-router-dom';
+import { Form, Typography, Input, Button, notification } from 'antd';
+import styled from 'styled-components';
+import qs from 'query-string';
+
+import { useAuth } from '../../hooks';
+import { UserAPI } from '../../api';
+import { validatePassword } from './Auth';
+
+function ResetPassword() {
+  const [form] = Form.useForm();
+  const { authenticating } = useAuth();
+  const [password, setPassword] = useState('');
+  const [confirmPassword, setConfirmPassword] = useState('');
```

Comment:
the name should not contain verbs. 
`confirmPassword` => function that confirms password
`passwordConfirmation` => variable with password confirmation